Add LLM Eval Instructions (#19897)
DataDog/integrations-core: anthropic/README.md

## Setup

# Configuring Anthropic LLM Evaluations for Datadog

## Overview

> Datadog's **LLM Observability** enables end-to-end monitoring of your LLM application using **Anthropic** models. Follow the steps below to configure your Anthropic integration for **LLM Evaluations**.

Reviewer: ```suggestion
Datadog's LLM Observability enables end-to-end monitoring of your LLM application using Anthropic models. Follow the steps below to configure your Anthropic integration for LLM Evaluations.
```

## Prerequisites

- An **Anthropic account** with access to model deployments.
- A **valid Anthropic API key** with **write permissions** for model capabilities.

> ## Setup Instructions

Reviewer: ```suggestion
## Setup
```

> ### 1. Generate an Anthropic API Key

Reviewer: ```suggestion
### 1. Generate an Anthropic API key
```

> 1. Login to your [Anthropic Dashboard][3].

Reviewer: ```suggestion
1. Login to your [Anthropic dashboard][3].
```

> 2. Navigate to **API Keys** under your profile.

Reviewer: ```suggestion
2. Navigate to **API keys** under your profile.
```
verified in anthropic UI

3. Click the **Create Key** button.

>    - For **LLM Observability**, ensure that the API key has **write** permission for **model capabilities** to allow Datadog to invoke models in your Anthropic account.

Reviewer: ```suggestion
   - For LLM Observability, ensure that the API key has **write** permission for **model capabilities**. This allows Datadog to invoke models in your Anthropic account.
```

4. Copy the created API key to your clipboard.

> ### 2. Configure Datadog's Anthropic Integration

Reviewer: ```suggestion
### 2. Configure Datadog's Anthropic integration
```

> 1. Navigate to the configuration tab inside Datadog Anthropic integration tile.

Reviewer: ```suggestion
1. Navigate to the configuration tab inside Datadog's Anthropic integration tile.
```

> 2. Enter an **account name** and **paste the Anthropic API key** in the **account configuration** in Datadog.

Reviewer: ```suggestion
2. On the **Configure** tab, click **Add New**.
3. Under **Name**, enter a name for your account. Under **API key**, enter your Anthropic API key.
4. Click the check mark to save.
```

> 3. To enable LLM Evaluations:

Reviewer: ```suggestion

### 3. Enable LLM Evaluations
```
seems like this is the one meant to be a numbered heading, not Additional Notes

>    - Go to **LLM Observability > Settings**.

Reviewer 2: ```suggestion
   - Go to **LLM Observability > Settings > Evaluations**.
```

>    - Go to **LLM Observability > Settings > Evaluations**.

Reviewer: ```suggestion
1. Go to **LLM Observability > Settings > Evaluations**.
```

>    - Add your **Anthropic API key** to evaluate and monitor your **LLM applications**.

Reviewer 2: ```suggestion
   - Select **Anthropic account** to evaluate and monitor your **LLM applications**.
```

>    - Select **Anthropic account** to evaluate and monitor your **LLM applications**.

Reviewer: Is this correct? I'm not seeing how you can select an Anthropic account on the LLM settings evaluations page

> ### 3. Additional Notes

Reviewer: ```suggestion
### Additional Notes
```

> - This integration allows **LLM Observability** to track **Anthropic model performance**.

Reviewer: ```suggestion
- This integration allows LLM Observability to track Anthropic model performance.
```

Also, is this note really necessary? seems pretty self-evident from the overview

> - **No additional permissions** are required beyond enabling write access for model capabilities.

Reviewer: ```suggestion
- No additional permissions are required beyond enabling write access for model capabilities.
```

## Setup

# Configuring Anthropic LLM Evaluations for Datadog

## Overview

Datadog's **LLM Observability** enables end-to-end monitoring of your LLM application using **Anthropic** models. Follow the steps below to configure your Anthropic integration for **LLM Evaluations**.

## Prerequisites

- An **Anthropic account** with access to model deployments.
- A **valid Anthropic API key** with **write permissions** for model capabilities.

## Setup Instructions

### 1. Generate an Anthropic API Key

> 1. Login to your [Anthropic Dashboard][10].

Reviewer 3: The link for this is missing here: https://github.com/DataDog/integrations-core/pull/19897/files#diff-ca4c7728f2929cd62aa0aabf305385711e71e9e0e3ec91d929f4af77723856e2R173-R174

2. Navigate to **API keys** under your profile.
3. Click the **Create Key** button.
   - For LLM Observability, ensure that the API key has **write** permission for **model capabilities**. This allows Datadog to invoke models in your Anthropic account.
4. Copy the created API key to your clipboard.

### 2. Configure Datadog's Anthropic integration

1. Navigate to the configuration tab inside Datadog's Anthropic integration tile.
2. On the **Configure** tab, click **Add New**.
3. Under **Name**, enter a name for your account. Under **API key**, enter your Anthropic API key.
4. Click the check mark to save.

### 3. Enable LLM Evaluations
1. Go to **LLM Observability > Settings > Integrations**.

>    - Select the **Anthropic account** that will be used evaluate and monitor your **LLM applications**.

Author: @davit-bar Can you please verify?

>    - Select the **Anthropic account** that will be used evaluate and monitor your **LLM applications**.

Reviewer 2: This is not necessary/mandatory step. This is optional step to opt-in to run evaluations for applications that use llm observability. For configuration/integration, not needed.
